Include/object.h

```diff
@@ -394,11 +394,11 @@ static inline void Py_SET_REFCNT(PyObject *ob, Py_ssize_t refcnt) {
         ob->ob_ref_shared = _Py_REF_SHARED(refcnt, _Py_REF_MERGED);
     }
 #endif  // Py_GIL_DISABLED
-#endif  // Py_LIMITED_API+0 < 0x030d0000
 }
 #if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 < 0x030b0000
```

Comment:
I'm surprised that the limited API included a macro call, but at least if the limited API isn't selected at all, the macro shouldn't be defined?
```suggestion
#if defined(Py_LIMITED_API) && Py_LIMITED_API+0 < 0x030b0000
```

Reply:
The cast to `PyObject*` is only done for non-limited API and for limited C API version 3.10 and older: see PEP 670 – Convert macros to functions in the Python C API: [Avoid the cast in the limited C API version 3.11](https://peps.python.org/pep-0670/#avoid-the-cast-in-the-limited-c-api-version-3-11).

Reply:
Ah! I understand, thank you.